### PR TITLE
Bump wasmparser to 0.78.0

### PR DESCRIPTION
**Pull request by fitzgen**

### CODE REVIEW

File: crates/wasmprinter/Cargo.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ Rust converter from the WebAssembly binary format to the text format.
 
 [dependencies]
 anyhow = "1.0"
-wasmparser = { path = '../wasmparser', version = '0.77' }
+wasmparser = { path = '../wasmparser' }
```
Comment (alexcrichton) [inline crates/wasmprinter/Cargo.toml:17]: Since we publish wasmparser we need to keep the dependency version here

**Review by alexcrichton — APPROVED** [review]
Mind bumping wasmprinter as well (minor version) so we can publish a version that pulls in the latest wasmparser too?

File: crates/wasmprinter/Cargo.toml (hunk 1 above)
Comment (fitzgen) [reply crates/wasmprinter/Cargo.toml:17]: Ah for some reason I thought that cargo would figure this out manually.

**Comment by fitzgen** [comment]
Tagged and published!

**Pull request merged**